packages/editor/src/client/atoms/typography.tsx

```diff
@@ -93,9 +93,9 @@ export function marginForTypographySpacing(spacing: TypographySpacing): string {
 }
 
 export interface TypographyVariantCSS {
```

Comment:
Looks like updating @types/react has also upgraded "csstypes", updated typings accordingly.
We probably should add it to the package.json so that this doesn't happen anymore, what do you think?

Reply:
Hmm, indeed if we use it directly we should lock it.